jib-gradle-plugin/src/integration-test/resources/gradle/projects/simple/build-extra-dirs.gradle

```diff
@@ -15,4 +15,4 @@ dependencies {
 }
 
 jib.to.image = System.getProperty("_TARGET_IMAGE")
-jib.extraDirectory = ['src/main/custom-extra-dir', 'src/main/custom-extra-dir2']
+jib.extraDirectories = ['src/main/custom-extra-dir', 'src/main/custom-extra-dir2']
```

Comment:
Do we need to have tests that use jib.extraDirectory directly in the build.gradle?

Reply:
I'll add one.